src/wagtail_live/models.py

```diff
@@ -133,6 +134,36 @@ def add_live_post(self, live_post):
         self.last_updated_at = post_created_at
         self.save(clean=False)
 
+        live_post = self.get_live_post_by_index(lp_index)
+        live_page_update.send(
+            sender=self.__class__,
+            channel_id=self.channel_id,
+            renders={
+                live_post.id: live_post.render(context={"block_id": live_post.id})
+            },
+            removals=[],
+        )
+
+    def update_live_post(self, live_post):
+        """Updates a live post when it has been edited.
+
+        Args:
+            live_post (livePostBlock):
+                Live post to update.
+        """
+
+        live_post.value["modified"] = self.last_updated_at = timezone.now()
+        self.save(clean=False)
```

Comment:
This is from another PR but I will change it in a next PR. This was done so that the `last_updated_at` field isn't changed when a post is added via a receiver.

Reply:
Ah, maybe you can achieve the same effect by doing `self.__class__.objects.filter(pk=self.pk).update(last_updatet_at=timezone.now()`? I'm not sure what is nicer, but I think this might be more failsafe. Maybe others have an opinion as well.

Reply:
I intend to remove `clean=False` in a following PR.